js/widgets/page.dialog.js

```diff
@@ -33,63 +33,83 @@
 
 return $.widget( "mobile.page", $.mobile.page, {
 	options: {
+		classes: {
+			"ui-page-dialog-close-button":
+			"ui-button ui-corner-all ui-button-icon-only",
+			"ui-page-dialog-close-button-icon": "ui-icon-delete ui-icon",
+			"ui-page-dialog-contain": "ui-overlay-shadow ui-corner-all"
+		},
 
 		// Accepts left, right and none
 		closeBtn: "left",
 		closeBtnText: "Close",
 		overlayTheme: "a",
-		corners: true,
 		dialog: false
 	},
 
 	_create: function() {
+		this._ui = {};
 		this._super();
 		if ( this.options.dialog ) {
-
-			$.extend( this, {
-				_inner: this.element.children(),
-				_headerCloseButton: null
-			} );
-
-			if ( !this.options.enhanced ) {
-				this._setCloseBtn( this.options.closeBtn );
+			if ( this.options.enhanced ) {
+				this._ui.wrapper = this.element.children();
```

Comment:
Im i'm understanding what your trying to do here in the enhanced case there should be only one child so this is not really a collection. However i don't like the assumption its definitely not clear what going on here. lets do something more explicit like `this._ui.wrapper = this.element.children( ".ui-page-dialog-contain" ).eq( 0 );`